scripts/scenarios/cloud8/qa/no-ssl/qa-scenario-2a.yaml

```diff
@@ -222,19 +208,6 @@ proposals:
       heat-server:
       - cluster:services
 
-- barclamp: ceilometer
```

Comment:
Why are you removing Ceilometer from the scenario?

Reply:
during prechecks if monasca is missing upgrade wont go further unless i remove ceilometer  and as such case already covered in qa-scenario-8a(deploying monasca) I decided remove it from here ,other option would be deploy monasca , but because not enough nodea avaible i think it would be complicated .

Reply:
Ah, ok. I was wondering about that because I created dedicated precheck clean scenarios in https://github.com/SUSE-Cloud/automation/pull/3682 which is a little...starved for reviews... :-)

What do we do about this? Merge the dedicated scenarios or remove all precheck breakers from the existing ones (in that case there'd be more things to change). I for one would prefer having dedicated upgrade scenarios in order to continue testing the barclamps that are still supported on Cloud 8 for Cloud 8 rather than disabling them for regular Cloud 8 (i.e. in non-upgrade situations) as well.

Reply:
I wanted do two things together fix broken scenario in general for SOC8 and make them working for upgrade so we dont need duplicate
I expect those scenarios under cloud8 run only for upgrade porpuse I dont think we would use them for anything else .

Reply:
Ok. Then I'll +1 this for now, since I have none of the non-precheck fixes in my pull request. We may need a follow-up to take care of Aodh, Trove and more Ceilometer occurences as well though (unless someone has taken care of that in the time since I created https://github.com/SUSE-Cloud/automation/pull/3682 ; haven't checked).